examples/gno.land/p/demo/dao_maker/havl/havl.gno

```diff
@@ -0,0 +1,133 @@
+package havl
```

Comment:
can you add more info about what is this package about?

it looks like something that should be located in `p/demo/havl`.

+ add some tests.